src/site/content/en/learn/css/layout/index.md

```diff
@@ -0,0 +1,401 @@
+---
+title: Layout
+description:
+authors:
+  - andybell
+date: 2021-04-20
+tags:
+  - css
+---
+
+# Layout
+
+Imagine you're working as a developer,
+and a designer colleague hands you a design for a brand new website.
+The design has all sorts of interesting layouts and compositions:
+two-dimensional layouts that are considerate of viewport width and height,
+as well as layouts that need to be fluid and flexible.
+How do you decide the best way to style these with CSS?
+
+CSS provides us with various ways to solve layout problems,
+both on a horizontal axis, on a vertical axis, or even both.
+Choosing the right layout method for a context can be hard,
+and often you may need more than one layout method to solve your problem.
+To help with this, over the next few lessons,
+you'll learn about the unique features of each CSS layout mechanism to inform those decisions.
+
+## Layout: a brief history
+
```

Comment:
I really think an image/diagram would help break up the blocks of text in these first two sections. Can we design some sort of timeline of layout features over time for this? i.e. CSS1, Table layout, Floats, Flexbox, Grid, Container Queries(!)

Reply:
@andy-piccalilli can you look at this when doing your images today?

Reply:
![scamp](https://user-images.githubusercontent.com/8672583/116418602-396ad300-a834-11eb-861e-a9bd1018a299.png)

Here's a dog-rough scamp in procreate. I worry with this, that floats will be seen as legacy, but really, they're a totally valid current layout mechanism.

Reply:
<img width="760" alt="timeline" src="https://user-images.githubusercontent.com/8672583/116528650-fceeb400-a8d3-11eb-814e-dcbdaa0be451.png">

think we're on to a winner here

Reply:
love it, can we add when floats were added since we mention them?  I believe it was [CSS1](https://www.w3.org/TR/REC-CSS1/) along with tables

Reply:
It was indeed and good shout!